prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    var $container = document.querySelector('#container');
+    console.log($container);
+
+    var $heading = document.createElement('h1');
+    console.log($heading);
+
+    var textContent = document.createTextNode('Hello World!');
```

Comment:
Hi Christina! Here you'll want to append the `textContent` to the `$heading` and then append the heading to the container instead of the text content